tests/Integration/Classes/Checkout/CheckoutAddressesStepTest.php

```diff
@@ -76,50 +76,46 @@ protected function setUp(): void
     public function testIfCustomerHasNoAddressesThenDeliveryAddressFormIsOpen(): void
     {
         $this->session->method('getCustomerAddressesCount')->willReturn(0);
-        $this->assertArraySubset(
-            [
-                'show_delivery_address_form' => true,
-            ],
+        $this->session->method('getCustomerAddressesCount')->willReturn(1);
```

Comment:
?

Reply:
?

Reply:
?

Reply:
I'm a @Progi1984 translator.
The `?` he writes was an interrogation about why you did:
```php
$this->session->method('getCustomerAddressesCount')->willReturn(0);
```
Then

```php
$this->session->method('getCustomerAddressesCount')->willReturn(1);
```

What's the aim behind this?

@Progi1984 Next time, give a little sentence to understand what you have in mind :wink:

Reply:
That's a bad copy/paste, my bad 😅